src/designSystem/components/CircleBadgeForm/CircleBadgeForm.css

```diff
@@ -5,16 +5,16 @@
   box-shadow: var(--base-shadow);
   cursor: pointer;
   display: flex;
-  height: 30px;
-  width: 30px;
+  height: 20px;
+  width: 20px;
 }
 
 .dm-screen-design-system-circle-badge-form input {
   background: none;
   border: none;
   color: var(--white);
   font-family: var(--base-font-family);
-  font-size: var(--sub-font-size);
+  font-size: 10px;
```

Comment:
Questioned making this a variable, but decided against it. If it comes up again, I'll make this a variable.